dir.props

```diff
@@ -99,6 +99,35 @@
 
   <Import Project="$(MSBuildThisFileDirectory)pkg/ExternalPackages/versions.props" Condition="Exists('$(MSBuildThisFileDirectory)pkg/ExternalPackages/versions.props')" />
 
+  <!-- Dependency auto-upgrade configuration -->
+  <ItemGroup>
+    <DependencyBuildInfo Include="CoreFx">
+      <RawUrl>https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master</RawUrl>
```

Comment:
Should we parameterize the branch in some way? At least I think we should pull out master and put it into its own property. So we can change it to release or dev/api in the other branches as appropriate.